libs/wingc/src/dtsify/mod.rs

```diff
@@ -416,8 +416,8 @@ pub interface Interface {
 }
 
 pub interface ClassInterface {
-	addHandler(handler: inflight (str): str);
-	inflight bar();
+	addHandler(handler: inflight (str): str): void;
+	inflight bar(): void;
```

Comment:
Should interfaces and classes be consistant here? Class methods allow ommiting the `void`